_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -61,7 +61,10 @@ class HealthCareApp extends React.Component {
             isWhite: false
           },
 
-          veteranAddress: {}
+          veteranAddress: {
+            homePhone: '555-555-5555',
+            mobilePhone: '555-555-5555'
```

Comment:
Just for my own OCDness, use a different number here. 1111 maybe? That way we can tell quickly if we've cross wires between the fields.